typescript/aleo-sdk/src/clients/base.ts

```diff
@@ -58,9 +65,13 @@ export class AleoBase {
       getOrInitConsensusVersionTestHeights(this.consensusVersionHeights);
     }
 
+    this.prefix = this.chainId ? TESTNET_PREFIX : MAINNET_PREFIX;
```

Comment:
why do we decide if it is a testnet or not based on if chainId is defined? shouldn't we inspect it's value to make a decision? If chainId is 0 is it ok to use the `MAINNET_PREFIX` ? I'd rather be explicit about the chainId -> prefix mapping

Reply:
we do a check in the constructor above for the only two allowed values (`+chainId === 0 || +chainId === 1`). Therefore I think we can leave it as is